lib/ansible/modules/cloud/amazon/redshift.py

```diff
@@ -244,6 +240,14 @@ def _collect_facts(resource):
             facts['public_ip_address'] = node['PublicIPAddress']
             break
 
+    # Some parameters are not ready instantly if you don't wait for available
```

Comment:
Should you default these values to None that way the returned keys are always the same? It would also be good to add this note to the return docs for these four.